### PR TITLE
Fixes #15781 - add ssl_cert_file to env check

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: checks/env_variables.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+
+variables = %w(http_proxy https_proxy ssl_cert_file)
+
+if variables.map{ |variable| ENV[variable] }.compact.any?
```
Comment (ekohl) [inline checks/env_variables.rb]: rubocop wants a space between ```map``` and ```{```

File: checks/env_variables.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  $stderr.puts "Please unset the following environment variables before running the installer: #{variables.join(', ')}"
+  exit 1
```
Comment (ekohl) [inline checks/env_variables.rb]: And trailing whitespace

File: checks/env_variables.rb (hunk 1 above)
Comment (jlsherrill) [inline checks/env_variables.rb]: updated

File: checks/env_variables.rb (hunk 2 above)
Comment (jlsherrill) [inline checks/env_variables.rb]: updated!

**Comment by jlsherrill** [comment]
Thanks for the review @ekohl !

**Pull request merged**